src/hooks/accounts/resolvers/query/activeAccount.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback } from 'react';
+import { Account } from '../../../../generated/graphql';
+import { GET_ACCOUNTS } from '../../queries/useGetAccountsQuery';
+import { usePersistActiveAccount } from '../../lib/usePersistActiveAccount';
+import { find } from 'lodash';
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client';
+import { withErrorHandler } from '../../../apollo/withErrorHandler';
+
+// make sure the __typename is well typed
+export const __typename: Account['__typename'] = 'Account';
+
+// helper function to decorate the extension entity for normalised caching
+const withTypename = (account: Account) => ({
+  __typename,
+  ...account,
+});
```

Comment:
This is used multiple times across various files. please extract to some utils so it can be shared.

Reply:
extracted to types.tsx